libkbfs/interfaces.go

```diff
@@ -942,12 +942,6 @@ type BlockOps interface {
 	Ready(ctx context.Context, kmd KeyMetadata, block Block) (
 		id BlockID, plainSize int, readyBlockData ReadyBlockData, err error)
 
-	// Put stores the readied block data under the given block
-	// pointer (which belongs to the TLF with the given ID) on the
-	// server.
-	Put(ctx context.Context, tlfID TlfID, blockPtr BlockPointer,
```

Comment:
Hmm, quota reclamation doesn't involve putting blocks, only removing references, which is a direct call to `MDServer`.  So I don't think having `MDOps.Put` would change things one way or the other.